FinanceTracker/FinanceTracker/Models/Account.cs

```diff
@@ -2,30 +2,26 @@
 {
     public class Account
     {
+        public List<Operation> Operations = new();
         public decimal Balance
         {
             get
             {
-                return Incomes - Expences;
-            }
-        }
-        public decimal Incomes { get; set; }
-        public decimal Expences { get; set; }
-
-        public Account()
-        {
-            Incomes = 0.0m;
-            Expences = 0.0m;
-        }
+                decimal delta = 0.0m;
+                foreach (var operation in Operations)
```

Comment:
Create method `AddOperation` and calculate delta there to avoid calculation on each read